extensions/notebook/src/jupyter/jupyterServerManager.ts

```diff
@@ -102,7 +102,11 @@ export class LocalJupyterServerManager implements nb.ServerManager, vscode.Dispo
 	}
 
 	private async doStartServer(): Promise<IServerInstance> {        // We can't find or create servers until the installation is complete
-		let installation = await this.options.jupyterInstallation;
+		let installation = this.options.jupyterInstallation;
+
+		await installation.promptForPythonInstall();
```

Comment:
Can we have this return false / reject if user doesn't do python install, and then reject this doStartServer function and handle farther up? Basically waiting forever on installReady (which never succeeds) is badness and we should cleanly fail out instead.

Reply:
Added the reject. Do you still want to wait on the kernel revert functioanlity?